packages/net-stubbing/lib/server/middleware/request.ts

```diff
@@ -19,14 +18,13 @@ import {
   getBodyEncoding,
 } from '../util'
 import { InterceptedRequest } from '../intercepted-request'
-import type { BackendRoute } from '../types'
 
-const debug = Debug('cypress:net-stubbing:server:intercept-request')
+// do not use a debug namespace in this file - use the per-request `this.debug` instead
+// available as cypress-verbose:proxy:http
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const debug = null
```

Comment:
We could removed this right?

Reply:
This is here so that developers get an error if they attempt to invoke `debug` in this file - since this file contains only `proxy` middleware, any `debug` calls should be to `this.debug` so they get the advantage of the extra contextual information (method, URL, stage) about the HTTP request: https://github.com/cypress-io/cypress/blob/2a67033222c5a9a2148fdaeac6e47f9a306c1b25/packages/proxy/lib/http/index.ts#L298-L302

The same pattern is followed in `proxy`: https://github.com/cypress-io/cypress/blob/2a67033222c5a9a2148fdaeac6e47f9a306c1b25/packages/proxy/lib/http/request-middleware.ts#L9-L12

I guess you could get by with a lint rule for this, too, but I think I did this pattern in `proxy` before I knew about blocking variable names with eslint.